### PR TITLE
Delete app flow

### PR DESCRIPTION
**Pull request by diogoperillo**

### CODE REVIEW

File: packages/launcher/src/renderer/launcher/apps/AppItem.js [hunk 1]
```diff
@@ -183,32 +233,69 @@ export default class AppItem extends Component<Props, State> {
...
             }
           />
-          {onUpdate && <UpdateBadge />}
+          {!installing && !editing && onUpdate && <UpdateBadge />}
+          <OverlayBaloon visible={deleting && editing}>
+            <Text variant="tooltipTitle">
+              Do you really want to delete {appName}?
+            </Text>
+            <Text variant="tooltipText">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin
```
Comment (PaulLeCam) [inline packages/launcher/src/renderer/launcher/apps/AppItem.js]: Lorem ipsum to replace or remove please.

File: packages/launcher/src/renderer/launcher/apps/AppItem.js [hunk 2]
```diff
@@ -254,6 +343,9 @@ const InstalledView = (props: InstalledProps) => {
 
   return (
     <AppItem
+      {...props}
+      editing={props.editing}
+      deleting={props.deleting}
```
Comment (PaulLeCam) [inline packages/launcher/src/renderer/launcher/apps/AppItem.js]: These 2 props should already be injected with the `...props` spread above.

File: packages/launcher/src/renderer/styles.css [hunk 3]
```diff
@@ -111,6 +112,19 @@ body,
...
 }
 
+.app-editing {
+  transition: all 0.3s;
+  transform: perspective(400px) rotateY(-18deg);
+  animation: pulse 0.8s infinite both;
+}
+
+@keyframes pulse {
+
```
Comment (PaulLeCam) [inline packages/launcher/src/renderer/styles.css]: White space is not consistent with the rest of the stylesheet, could you format it please?

**Comment by PaulLeCam** [comment]
@diogoperillo any update on this please? Seems like the changes to the CircleCI config are messing things up.

**Comment by diogoperillo** [comment]
Should be good now @PaulLeCam

**Pull request merged**